docs/contributors.rst

```diff
@@ -69,6 +69,11 @@ For a debug build with debug symbols, checked assertions etc., use the extra fla
 
     $ HOROVOD_WITH_PYTORCH=1 HOROVOD_WITH_TENSORFLOW=1 pip install --global-option build_ext --global-option --debug -v -e .
 
+To uninstall Horovod installed in dev mode, run the below command from the Horovod root directory:
+
+.. code-block:: bash
+
+    $ python setup.py develop -u
```

Comment:
How does this relate to `pip install .[dev,test]` above? Should we mention either or both?

Reply:
@EnricoMi this uninstall method is when you install the package in dev mode `-e` since the standard `pip uninstall` does not work in that case.

The part about running `pip install .[dev,test]` is a bit redundant. Should we just remove it and modify line 56 with:
```
HOROVOD_WITH_PYTORCH=1 HOROVOD_WITH_TENSORFLOW=1 pip install -v -e .[dev,test]
```
So when installing in dev mode, it checks for dev and test dependencies?

Reply:
I misread your comment in the changes, `python setup.py develop -u` is to uninstall an earlier dev installation. What is wrong about `pip uninstall horovod`?

Reply:
The `[dev]` adds a specific version for all ML frameworks to the dependencies and `[test]` adds test-specific dependencies as well. So either users install the frameworks in the desired version or simply use `[dev]` to get recommended versions.

Reply:
`pip uninstall horovod` did not work with the editable mode. But i just tried it using a virtual environment (instead of just the system environment), and it works. I will update the doc since we recommend a venv.
I can mention the requirements and remove the first call to `pip install`.